Copilot for Xcode.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -27,6 +27,15 @@
         "version" : "1.0.2"
       }
     },
+    {
+      "identity" : "generative-ai-swift",
```

Comment:
What do these changes do?

Reply:
The Package.resolved file is updated because I mostly develop in another workspace. Maybe it's better to just uncheck the file for this pull request if no dependency is added?